rca/utils/models.py

```diff
@@ -437,7 +581,15 @@ def legacy_news_and_events(self):
         ``page.legacy_news_and_events`` can be referenced in the
         template multiple times without triggering another
         cache lookup.
+
+        If there is no ``page.legacy_news_and_event_tags`` value, it returns
+        Editorial and Event pages with relationships to this page via
+        utils.NewsAndEventsMixin.
         """
+
+        if not self.legacy_news_and_event_tags.exists():
+            return NewsAndEventsMixin(page=self).get_data()
```

Comment:
praise: neat, I like the forethought here, using the new functionality from the start, but keeping it separate, rather than mashing it into the old mixin and having to unpick it later